server/app/views/applicant/ApplicantLayout.java

```diff
@@ -550,30 +550,46 @@ protected Optional<DivTag> maybeRenderBackToAdminViewButton(
   }
 
   /**
-   * Returns true if the request object points to a URI that is the Trusted Intermediary Dashboard.
-   * When a TI is impersonating an applicant to apply for them, this method will return false.
+   * Returns true if the request object points to a URI that is the Trusted Intermediary Dashboard
+   * or any of its related pages. When a TI is impersonating an applicant to apply for them, this
+   * method will return false.
    */
   private static boolean onTiDashboardPage(Http.Request request) {
     String currentPath = null;
-    String tiDashboardPath = null;
+
+    String tiDashboardPath =
+        controllers.ti.routes.TrustedIntermediaryController.dashboard(
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty())
+            .url();
+
+    String editClientPath =
+        controllers.ti.routes.TrustedIntermediaryController.editClient(1L).url();
+
+    // Remove the last two path segments to get the common path prefix "/admin/tiGroups"
```

Comment:
Question (non-blocking): It looks like `controllers.admin.TrustedIntermediaryManagementController.index(request: Request)` corresponds to just `"/admin/tiGroups", can we use that route here?

Reply:
Hm, we could, although that controller is for a completely different part of the application (it's for when CF Admins are managing the TI groups).  This is making me think that a better course of action would be to rename the TI client edit and add routes to "/admin/tiDash/..." given that they are subpages of the TI dashboard.  I think I'll do that.  Let me know if you see any issue with it.